src/status_im/ui/screens/wallet/request/events.cljs

```diff
@@ -10,4 +10,11 @@
                   [:navigate-to-clean :chat-list]
                   [:chat-with-command whisper-identity :request
                    {:contact contact
-                    :amount (:amount request-transaction)}]]}))
\ No newline at end of file
+                    :amount (:amount request-transaction)}]]}))
+
+(handlers/register-handler-fx
+  :wallet-validate-request-amount
+  (fn [{{:keys [web3] :wallet/keys [request-transaction] :as db} :db} _]
+    (let [amount (:amount request-transaction)
+          error (wallet.db/getAmountValidationError amount web3)]
```

Comment:
wouldn't it be clearer if getAmountValidationError (why is it camel and not kebab case btw ?) returned nil for the good cases ?

Reply:
done